component-template/{{ cookiecutter.slug }}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--
+Thank you for your pull request. Please provide a description above and
+review the checklist below.
+
+Contributors guide: ./CONTRIBUTING.md
+-->
+
+## Checklist
+<!--
+Remove items that do not apply. For completed items, change [ ] to [x].
+-->
+
+- [ ] keep pull requests small so they can be easily reviewed.
```

Comment:
Can we remove the periods in the list?

Reply:
In ensured they all are proper sentences instead.